src/internal.rs

```diff
@@ -37,6 +41,58 @@ impl Default for FileTypes {
     }
 }
 
+enum SizeLimitType {
+    Max,
+    Min,
+}
+
+pub struct SizeFilter {
+    size: u64,
+    limit_type: SizeLimitType,
+}
+
+impl SizeFilter {
+    pub fn is_within(&self, size: u64) -> bool {
+        match self.limit_type {
+            SizeLimitType::Max => size <= self.size,
+            SizeLimitType::Min => size >= self.size,
+        }
+    }
+}
+
+const KILO: u64 = 1024;
```

Comment:
Concerning units, I would like to follow the [official SI and EIC standards](https://en.wikipedia.org/wiki/Kibibyte). The "k", "M", "G", "T"... prefixes are for powers of 10 while the "Ki", "Mi", "Gi", "Ti"... prefixes are for powers of 2. For example:

- 1 MB = 10^6 byte = 1000² byte
- 1 MiB = 2^20 byte = 1024² byte

We can add the "Ki", "Mi".. prefixes later, but these numbers here should be `1000`, not `1024`.